oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticConnectionRule.java

```diff
@@ -70,62 +73,22 @@ public ElasticConnectionRule(String elasticConnectionString) {
-                    .withCommand("bash /tmp/elasticstartscript.sh")
-                    .withNetwork(network);
-            elastic.start();
-
+            elastic = ElasticTestServer.getESTestServer().container();
             setUseDocker(true);
         }
         return s;
     }
 
     @Override
     protected void after() {
-        if (elastic != null && elastic.isRunning()) {
-            elastic.stop();
-        }
-    }
-
-    private void downloadSimilaritySearchPluginIfNotExists(String localPluginPath, String pluginVersion) {
-        File pluginFile = new File(localPluginPath);
-        if (!pluginFile.exists()) {
-            LOG.info("Plugin file {} doesn't exist. Trying to download.", localPluginPath);
-            try (CloseableHttpClient client = HttpClients.createDefault()) {
-                HttpGet get = new HttpGet("https://github.com/alexklibisz/elastiknn/releases/download/" + pluginVersion
-                        + "/elastiknn-" + pluginVersion + ".zip");
-                CloseableHttpResponse response = client.execute(get);
-                InputStream inputStream = response.getEntity().getContent();
-                MessageDigest messageDigest = MessageDigest.getInstance("SHA-256");
-                DigestInputStream dis = new DigestInputStream(inputStream, messageDigest);
-                FileOutputStream outputStream = new FileOutputStream(pluginFile);
-                IOUtils.copy(dis, outputStream);
-                messageDigest = dis.getMessageDigest();
-                // bytes to hex
-                StringBuilder result = new StringBuilder();
-                for (byte b : messageDigest.digest()) {
-                    result.append(String.format("%02x", b));
-                }
-                if (!PLUGIN_DIGEST.equals(result.toString())) {
-                    String deleteString = "Downloaded plugin file deleted.";
-                    if (!pluginFile.delete()) {
-                        deleteString = "Could not delete downloaded plugin file.";
-                    }
-                    throw new RuntimeException("Plugin digest unequal. Found " + result + ". Expected " + PLUGIN_DIGEST + ". " + deleteString);
-                }
-            } catch (IOException | NoSuchAlgorithmException e) {
-                throw new RuntimeException("Could not download similarity search plugin", e);
+        ElasticConnection esConnection = useDocker() ? getElasticConnectionForDocker() : getElasticConnectionFromString();
+        if (esConnection != null) {
+            try {
+                esConnection.getClient().indices().delete(new DeleteIndexRequest(esConnection.getIndexPrefix() + "*"), RequestOptions.DEFAULT);
+                esConnection.close();
+            } catch (IOException e) {
+                LOG.error("Unable to delete indexes with prefix {}", esConnection.getIndexPrefix());
```

Comment:
For those cases (the ones that extend ElasticAbstractQueryTest), the delete call in ElasticConnectionRule would just be one extra unwanted call (since it would effectively be NOOP)